app/assets/stylesheets/custom.scss

```diff
@@ -129,20 +151,24 @@ td {
   background-color: rgba(255, 255, 255, .5);
 }
 
-// flash alerts
-#flash_error {
-  color: #ff0000;
+th {
   text-align: center;
 }
 
-#flash_alert {
-  color: #ff0000;
-  text-align: center;
+// complete loan message
+#complete-loan {
+  font-size: 20px;
```

Comment:
Properties should be ordered display, font-size, text-decoration, text-decoration-color